beta/session-3.md

```diff
@@ -13,11 +13,11 @@ Tags help you assign human-friendly names to specific commits. They can also be
 
 To create a tag, identify your last commit id with the following:
 
-    liquibase data log --repository=<your-repos-name>
+    liquibase data log --repository=myrepos
 
 Once you have the commit id, apply a tag with the following command:
 
-    liquibase data datatag --repository=<your repos name> --commit=<commit id> --tags=<tag name>
```

Comment:
This should just be `liquibase data tag`.

Reply:
fixed!